etc/ci/scenario/servo_speedometer.py

```diff
@@ -118,6 +120,7 @@ def run():
 
 
 if __name__ == "__main__":
-    result = run()
-    if not result:
-        sys.exit(1)
+    with HarmonyDevicePerfMode():
```

Comment:
I think the location might be important. I think it needs to be after we start the app, since that unlocks the screen (which this command doesn't do yet).